src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-form/rbd-snapshot-form.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="modal-header">
   <h4 class="modal-title float-left"
-      i18n>{ editing, select, true {Rename} other {Create}} RBD Snapshot</h4>
+      i18n="form title|Example: Create rbdSnapshot@@formTitle">{{ action | titlecase }} {{ resource | upperFirst }}</h4>
```

Comment:
We are already translating the `action` and `resource` in the component constructor, so what is the reason to use the `i18n` tag here?

Reply:
@ricardoasmarques As in the other forms, I provide a custom id with description:
https://angular.io/guide/i18n#use-a-custom-id-with-a-description
If we agree that this is not needed anymore, we can create a PR to remove this and all the similar cases.

Reply:
Thanks for the explanation